client/lib/plans/constants.js

```diff
@@ -54,12 +73,28 @@ export const plansList = {
 			FEATURE_FREE_THEMES,
 			FEATURE_3GB_STORAGE,
 			FEATURE_COMMUNITY_SUPPORT
-		]
+		],
+		getBillingTimeFrame: () => i18n.translate( 'for life' )
+	},
+
+	[ PLAN_PERSONAL ]: {
+		getTitle: () => i18n.translate( 'Personal' ),
```

Comment:
In 4b735a13378a1977b8a7e5c45b5dbd2964a6d8a2 I intended to do the same, but I wanted to keep everything consistent, so I imported the constants from `personal-plan.js`.

It turned out that JUST `import { personalPlan } from './personal-plan';` is making tests fail:
- `state plans reducer #items() should load persisted state:` 
- `state plans actions #requestPlans() - success should dispatch RECEIVE action when request completes`

We probably should look into why that happens, we dont want personal plan failing...